src/api/lib/tasks/dev.rake

```diff
@@ -72,15 +72,31 @@ namespace :dev do
   desc 'Run all linters we use'
   task :lint do
     Rake::Task['haml_lint'].invoke
-    Rake::Task['dev:lint:ruby'].invoke
+    Rake::Task['dev:lint:rubocop:all'].invoke
     sh 'jshint ./app/assets/javascripts/'
     Rake::Task['db:structure:verify'].invoke
     Rake::Task['db:structure:verify_no_bigint'].invoke
   end
   namespace :lint do
-    desc 'Run the ruby linter'
-    task :ruby do
-      sh 'rubocop -D -F -S --fail-level convention ../..'
+    namespace :rubocop do
+      task all: [:contrib, :dist, :rails] do
```

Comment:
this is not what we want. We want to run it twice: one for the rails directory and one for the rest. Only the Rails directory is special, splitting the rest is extra work without no reason. It is what is in the card as well.

Reply:
Seems we could add the '--ignore-parent-exclusion' parameter to make this work. @eduardoj let's try that tomorrow

Reply:
@bgeuken 

> Seems we could add the '--ignore-parent-exclusion' parameter to make this work. 

exactly... and even without that you could rename the files... :una